tests/acc_test.go

```diff
@@ -1216,10 +1217,30 @@ func Test_Dependencies(t *testing.T) {
 					},
 				},
 			}).Equal(t, r.Inputs)
-			autogold.Expect([]urn.URN{}).Equal(t, r.Dependencies)
+			//nolint:lll
+			autogold.Expect([]urn.URN{urn.URN("urn:pulumi:test::ts-dep-tester::random:index/randomInteger:RandomInteger::seed")}).Equal(t, r.Dependencies)
 			autogold.Expect(map[resource.PropertyKey][]urn.URN{
-				resource.PropertyKey("__module"):     {},
-				resource.PropertyKey("moduleInputs"): {},
+				resource.PropertyKey("__module"): {},
+				//nolint:lll
+				resource.PropertyKey("moduleInputs"): {urn.URN("urn:pulumi:test::ts-dep-tester::random:index/randomInteger:RandomInteger::seed")},
+			}).Equal(t, r.PropertyDependencies)
+		}
+
+		// TODO is this enough for dependent to depend on randmod:index:Module that itself does not seem to
```

Comment:
This feels a bit sketchy.. I am wondering if transitive deps actually do not work as yet.